scripts/core/editor3/components/media/MediaBlock.tsx

```diff
@@ -93,11 +94,11 @@ export class MediaBlockComponent extends React.Component<any, any> {
      * @name MediaBlockComponent#onChange
      * @description Triggered (debounced) when the image caption input is edited.
      */
-    onChange({target}) {
+    onChange(value, placeholder) {
         const {block, changeCaption} = this.props;
         const entityKey = block.getEntityAt(0);
 
-        changeCaption(entityKey, target.value, target.placeholder);
+        changeCaption(entityKey, value, placeholder);
```

Comment:
it's a placeholder, but now that you mention it, it's being used as a `field` value [here](https://github.com/superdesk/superdesk-client-core/blob/1543083b8c9481406cd461bb4589e27c2c9c0060/scripts/core/editor3/reducers/editor3.tsx#L429-L433) which doesn't make sense as this value is coming from `gettext`. I'm thinking that if the field is being translated this code wouldn't work! Or I'm missing something

Reply:
The comparison would work if you made a function to return that `Title` string translated and used the value from that function in both places.

I don't get through, why is there any logic based on a placeholder value?

Reply:
it seems we use the same reducer `chageImageCaption` to change the description or the title. I don't know why we're using the translated caption instead of just the key value `description_text` or `headline`.

Reply:
Looks like a mistake. I would replace it with a constant like IMAGE_TITLE/IMAGE_DESCRIPTION